app/controllers/admin/participants_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Admin
+  class ParticipantsController < Admin::BaseController
+    # TODO: probably add this back
```

Comment:
I agree, add them in `load_participant` - `authorize @participant` would be great there.

If you want it on a separate PR, cause we don't want to bother adding a policy here, feel free to do it there, maybe with a ticket number? Although this PR feels like it makes sense.

Reply:
ooops this was supposed be a _pre-commit_ todo, not one that gets pushed to github 😆